### PR TITLE
Page direction

### PR DESCRIPTION
**Pull request by sarracini**

### CODE REVIEW

**Comment by sarracini** [comment]
@k88hudson @oyiptong r?

**Comment by k88hudson** [comment]
I think since we're in firefox only `:-moz-dir(rtl)` is _ok_ but a bit dangerous to enforce this strategy, since you have to remember to do it every time you add a margin-left etc. etc.

We might want to consider using something like this instead, and have a compile step: https://www.npmjs.com/package/cssjanus

What do you think?

**Comment by k88hudson** [comment]
The other stuff you have in this patch is :+1:, I'm just wondering about the css strategy

File: src/components/Search/Search.scss [hunk 1]
```diff
@@ -18,11 +18,18 @@
...
     margin: 0;
-    border-radius: 3px 0 0 3px;
+    border-radius: 2px 0 0 2px;
+    font-size: 16px;
+    text-align: left;
     &:focus {
       outline: none;
       border-color: $focus-blue;
     }
+    &:-moz-dir(rtl) {
```
Comment (marcoscaceres) [inline src/components/Search/Search.scss:28]: Do we still need `-moz-dir`?

Comment (k88hudson) [inline src/components/Search/Search.scss:28]: i'm pretty only Firefox has implemented this selector, so yeah

**Comment by k88hudson** [comment]
@sarracini let's just squash/merge for now and file a follow-up for using a different RTL strategy

**Comment by k88hudson** [comment]
R+

**Comment by sarracini** [comment]
Filed follow-up: https://github.com/mozilla/remote-newtab/issues/189

**Pull request merged**